engine/src/main/java/org/terasology/logic/console/Console.java

```diff
@@ -59,6 +59,22 @@
      */
     void addMessage(Message message);
 
+    /**
+     * Adds a message to the console (as a CoreMessageType.CONSOLE message)
+     *
+     * @param message
+     * @param newLine Flag indicates about new line at the end of message
+     */
+    void addMessage(String message, boolean newLine);
+
+    /**
+     * Adds a message to the console (as a CoreMessageType.CONSOLE message)
+     *
+     * @param message
+     * @param type
+     * @param newLine Flag indicates about new line at the end of message
```

Comment:
Same as earlier comment. Also, the parameter "type" could use some explanation. What's it for?